data/models/canteen.py

```diff
@@ -110,7 +110,9 @@ def validate_siret(siret, field_name):
         Performs length and Luhn validation
         (https://portal.hardis-group.com/pages/viewpage.action?pageId=120357227)
         """
-        if siret is not None and len(siret) != 14:
+        if siret is None:
```

Comment:
```suggestion
        if siret is None or siret == "":
```
Quand on fait la modification dans `/admin` le siret est `""` (mais le siret_central_producer est None)

Reply:
Merci !